src/site/resources/styleguides/google-java-style-20220203/javaguide.html

```diff
@@ -1187,32 +1312,32 @@ <h3 id="s7.2-summary-fragment">7.2 The summary fragment</h3>
 <p class="tip"><strong>Tip:</strong> A common mistake is to write simple Javadoc in the form
 <code class="badcode prettyprint lang-java">/** @return the customer ID */</code>. This is
 incorrect, and should be changed to
-<code class="prettyprint lang-java">/** Returns the customer ID. */</code>.</p>
+<code class="prettyprint lang-java">/** Returns the customer ID. */</code> or
+<code class="prettyprint lang-java">/** {@return the customer ID} */</code>.</p>
```

Comment:
just extension of exampes, no action for us
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment